ui/src/app/views/workspace-wrapper/component.ts

```diff
@@ -69,7 +66,18 @@ export class WorkspaceWrapperComponent implements OnInit, OnDestroy {
         // do not render the child component with a stale workspace.
         this.workspace = undefined;
         this.accessLevel = undefined;
-        return this.workspaceStorageService.getWorkspace(ns, wsid);
+        // This needs to happen for testing. Otherwise it tries to make an api
```

Comment:
I hit this as well. Seeding the urlParamSTore with `{}` in the tests is the root of the issue - we may need a better story there eventually. I wouldn't say we *need* to deal with the issue in production code (but certainly this is more convenient).

Reply:
I will clarify the comment.